package.json

```diff
@@ -132,7 +132,7 @@
     "lru": "3.1.0",
     "lunr": "2.3.3",
     "marked": "0.5.0",
-    "mini-css-extract-plugin-with-rtl": "0.1.3",
+    "mini-css-extract-plugin-with-rtl": "github:jsnajdr/mini-css-extract-plugin-with-rtl",
```

Comment:
can we either publish this fork to npm under a new name or at least move the github repo under Automattic? :)

Reply:
Move to the A8c org in https://github.com/Automattic/wp-calypso/pull/26820/commits/e16fb8ec86609e24fa05f7246c9ae7b9f2b98d4f

`mini-css-extract-plugin-with-rtl-and-webpack-attr` is too long a name :)